nilearn/tests/test_signal.py

```diff
@@ -296,7 +296,7 @@ def test_clean_confounds():
                                       detrend=True, standardize=False)
     coeffs = np.polyfit(np.arange(cleaned_signals.shape[0]),
                         cleaned_signals, 1)
-    assert_true((abs(coeffs) < 10. * eps).all())  # trend removed
+    assert_true((abs(coeffs) < 100. * eps).all())  # trend removed
```

Comment:
>Or options: the norm of residuals is really related to the norm of what goes in. Relative errors are what it meaningful, not absolute. Thus, it is normal that it goes up.

for me this is the reason we need to relax both tests. To be convinced, rerun the same tests on the master with standardize=True (which results in a multiplication of the output signal by some factor) and the tests will not work.